client/views/curatorInbox/sourceIncidentReportsHeader.jade

```diff
@@ -0,0 +1,22 @@
+template(name="sourceIncidentReportsHeader")
+  .curator-source-details--sub-header
+    button.btn.btn-sm.btn-danger.delete(
+      disabled="{{#unless incidentsSelectedCount}} true {{/unless}}")
+      if deleting
```

Comment:
Can we take advantage of the latency compensation here since the UI changes depend on an update to the local collection. Should we remove the incidents from the local collection before the method call assuming there will be no error?